robot/impl/local_robot_test.go

```diff
@@ -1888,13 +1888,13 @@ func TestConfigPackages(t *testing.T) {
 func TestConfigPackageReferenceReplacement(t *testing.T) {
 	ctx := context.Background()
 	logger := golog.NewTestLogger(t)
-
+	// N.B. if the
```

Comment:
Note sure what this means, or if it was just left by accident.

Reply:
oops left by accident